.gitignore

```diff
@@ -33,3 +33,6 @@ yarn-error.log*
 # typescript
 *.tsbuildinfo
 next-env.d.ts
+
+#env
+.env
```

Comment:
Cool. It looks like we're using a .env file now. Can you update the README to add any env variables required to run the application?

I also forgot to do this when I added the meetup and settings services. If you don't mind, can you include these env variables in your README update? 🙏🏽 

```
MEETUP_API_BASE_URL=https://www.meetup.com
SETTINGS_BASE_URL=https://raw.githubusercontent.com/CodeandCoffeeCommunity/Code-and-Coffee-Website-Service-Settings/production/settings
```